### PR TITLE
ENH 2020 in review

### PR DESCRIPTION
**Pull request by beckermr**
## how to add a blog post

1. Put your post in `.rst` format in the `posts` directory. Look at the other posts for the format. 
   Make sure you have a `.. post::` directive at the top and a section title at the top.
2. You can build your post via the commands `make clean`, `ablog build`, and `ablog serve`.
3. Checkout the results in your browser.
4. When you are ready, make a PR!

### CODE REVIEW

**Comment by beckermr** [comment]
cc @conda-forge/core for comments and review!

**Review by mwcraig — APPROVED** [review]

**Review by kkraus14 — APPROVED** [review]

File: posts/2020-12-22-year-in-review.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- We've grown our active community, adding seven new members to the ``conda-forge`` Core team and at least two members to
+  the ``staged-recipes`` team.
+- We now have over 2,500 recipe maintainers in the ``conda-forge`` GitHub organization.
+
+Big New Features
+----------------
+
+We've shipped many big updates to our core infrastructure this year. These updates include the move to ``R`` 4.0, the new
+CentOS 7 ``sysroot`` for our compilers, completing the ``Python`` 3.8 migration, and the move of our compiler versions to ``GCC`` 9
+and ``clang`` 11. Finally, we've added our first completely cross-compiled platform, ``osx-arm64``, targeting Apple Silicon.
```
Comment (isuruf) [inline posts/2020-12-22-year-in-review.rst]: - [x] PyPy
- [x] standalone windows
- [x] miniforge

Comment (isuruf) [reply posts/2020-12-22-year-in-review.rst]: - [x] Removed python 2 support

**Comment by beckermr** [comment]
I think I got everything @isuruf. Thank you!

File: posts/2020-12-22-year-in-review.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+==============
+
+As 2020 winds down, the Core team thought it'd be fun to review some of the big accomplishments our community has made this year.
+
+
+Strong Growth
+-------------
+
+The ``conda-forge`` community has grown immensely this year. Here are some numbers to help give you an idea of the
+scale of our growth
```
Comment (beckermr) [inline posts/2020-12-22-year-in-review.rst]:
```suggestion
scale of our growth.
```

File: posts/2020-12-22-year-in-review.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+- **compiler upgrades**: We upgraded our compiler infrastructure to ``GCC`` 9 and ``clang`` 11.
+- **CentOS 7 and CentOS 6 EOL**: We shipped an option to enable our compilers to use the CentOS 7 ``sysroot`` in preparation
+  for the CentOS 6 EOL. We hope to complete the move to CentOS 7 early next year.
+- **miniforge**: We built our own standalone, ``miniconda``-like installers. These support a broad range of platforms, including
+  ``osx-arm64`` and ``linux-aarch64``.
+- **standalone Windows stack**: We fully decoupled our Windows recipes from the ``defaults`` channel by rebuilding the ``msys2``
+  recipes.
+- **Apple silicon support**: We added support for Apple silicon with our ``osx-arm64`` platform. This platform is our first
+  one to use a fully cross-compiled infrastructure.
+
```
Comment (isuruf) [inline posts/2020-12-22-year-in-review.rst:51]:
```suggestion
- **CUDA support**: We added support for building CUDA packages on windows and added CUDA 11.0 support.
```

File: posts/2020-12-22-year-in-review.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+As 2020 winds down, the Core team thought it'd be fun to review some of the big accomplishments our community has made this year.
+
+
+Strong Growth
+-------------
+
+The ``conda-forge`` community has grown immensely this year. Here are some numbers to help give you an idea of the
+scale of our growth
+
+- The community has added 3751 new, unique ``conda`` packages this year, along with a corresponding number of new feedstocks.
```
Comment (beckermr) [inline posts/2020-12-22-year-in-review.rst]:
```suggestion
- The community has added 3,751 new, unique ``conda`` packages this year, along with a corresponding number of new feedstocks.
```

File: posts/2020-12-22-year-in-review.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+Strong Growth
+-------------
+
+The ``conda-forge`` community has grown immensely this year. Here are some numbers to help give you an idea of the
+scale of our growth.
+
+- The community has added 3751 new, unique ``conda`` packages this year, along with a corresponding number of new feedstocks.
+- For the majority of 2020, the ``conda-forge`` channel on ``anaconda.org`` exceeded 100 million downloads per month.
+- In July of 2020, the ``conda-forge`` channel passed 2 billion total, all-time downloads.
+- We've grown our active community, adding seven new members to the ``conda-forge`` Core team and at least two members to
```
Comment (beckermr) [inline posts/2020-12-22-year-in-review.rst]:
```suggestion
- We've grown our developer community, adding seven new members to the ``conda-forge`` Core team and at least two members to
```

File: posts/2020-12-22-year-in-review.rst [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- **compiler upgrades**: We upgraded our compiler infrastructure to ``GCC`` 9 and ``clang`` 11.
+- **CentOS 7 and CentOS 6 EOL**: We shipped an option to enable our compilers to use the CentOS 7 ``sysroot`` in preparation
+  for the CentOS 6 EOL. We hope to complete the move to CentOS 7 early next year.
+- **miniforge**: We built our own standalone, ``miniconda``-like installers. These support a broad range of platforms, including
+  ``osx-arm64`` and ``linux-aarch64``.
+- **standalone Windows stack**: We fully decoupled our Windows recipes from the ``defaults`` channel by rebuilding the ``msys2``
+  recipes.
+- **Apple silicon support**: We added support for Apple silicon with our ``osx-arm64`` platform. This platform is our first
+  one to use a fully cross-compiled infrastructure.
+- **CUDA support**: We added support for building CUDA packages on windows and added CUDA 11.0 support.
```
Comment (beckermr) [inline posts/2020-12-22-year-in-review.rst:50]:
```suggestion
- **CUDA support**: We added support for building CUDA packages on windows and added CUDA 11.0 support.

```

File: posts/2020-12-22-year-in-review.rst [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+Strong Growth
+-------------
+
+The ``conda-forge`` community has grown immensely this year. Here are some numbers to help give you an idea of the
+scale of our growth.
+
+- The community has added 3,751 new, unique ``conda`` packages this year, along with a corresponding number of new feedstocks.
+- For the majority of 2020, the ``conda-forge`` channel on ``anaconda.org`` exceeded 100 million downloads per month.
+- In July of 2020, the ``conda-forge`` channel passed 2 billion total, all-time downloads.
+- We've grown our developer community, adding seven new members to the ``conda-forge`` Core team and at least two members to
```
Comment (beckermr) [inline posts/2020-12-22-year-in-review.rst]:
```suggestion
- We've grown our core developer community, adding seven new members to the ``conda-forge`` Core team and at least two members to
```

**Review by ocefpaf — APPROVED** [review]

**Review by xhochy — APPROVED** [review]

**Pull request merged**